lib/src/chart/bar_chart/bar_chart_painter.dart

```diff
@@ -358,13 +359,8 @@ class BarChartPainter extends AxisChartPainter<BarChartData> {
           tp.layout(maxWidth: getExtraNeededHorizontalSpace(holder));
           x -= tp.width + leftTitles.margin;
           y -= tp.height / 2;
-          canvasWrapper.save();
-          canvasWrapper.translate(x + tp.width / 2, y + tp.height / 2);
-          canvasWrapper.rotate(radians(leftTitles.rotateAngle));
-          canvasWrapper.translate(-(x + tp.width / 2), -(y + tp.height / 2));
-          y -= translateRotatedPosition(tp.width, leftTitles.rotateAngle);
-          canvasWrapper.drawText(tp, Offset(x, y));
-          canvasWrapper.restore();
+          x += calculateRotationOffset(tp.size, leftTitles.rotateAngle).dx;

```

Comment:
I like these changes. makes our code simpler.
May I ask you to do this for all of our chart painters? (Wherever we duplicated this logic)

Reply:
Unfortunately, i won't be able to do it with this pull request but possibly in the future.